app/src/main/assets/config.json

```diff
@@ -32,8 +32,34 @@
+●	Full name
+●	Gender
+●	Date of birth
+●	Home address
+●	Email address
+●	Contact numbers
+●	Health information – height, weight, blood type, blood pressure, medical history, allergy information, and other diagnostic information as required
+Use
+Your medical records and other personal data will be processed in compliance with the Data Privacy Act of 2012, and its Implementing Rules and Regulations. Your information is used for diagnosing and treating you. Your contact information may be used to send you reminders, prescription information, and other related communication.
+In addition, your personal information will be anonymized for use in research, disease monitoring, and analysis to help us improve our products and services. Anonymization removes your identifying information and protects your privacy.
+We may disclose your personal information:
+●	as required by law, such as to comply with a subpoena, or similar legal process;
+●	when we believe in good faith that disclosure is necessary to protect our rights, protect your safety or the safety of others, investigate fraud, or respond to a government request;
+●	with our trusted services providers who work on our behalf, do not have an independent use of the information we disclose to them, and have agreed to adhere to our privacy policy.
+●	if Intelehealth, Inc. is involved in a merger, acquisition, or sale of all or a portion of its assets, you will be notified via email and/or a prominent notice on our website of any change in ownership or uses of this information, as well as any choices you may have regarding this information.
+Protection Measures
+We provide physical, electronic, and procedural safeguards to protect information we process and maintain. 
+Only authorized personnel have access to your personal information. If a transfer of information is required, the exchange of which will be facilitated through email or hard copy. Digital files will be encrypted and/or password-protected before such transfers.
+Personal data records will be stored in a secure database for the period of our contract to provide these services, after which any physical records shall be disposed of through shredding, while digital files shall be anonymized. 
+Access and Correction
+You have the right to ask for a copy of any personal information we hold about you, as well as to ask for it to be corrected if you think it is wrong.
+If you’d like us to delete your personal data that you have provided to us, please contact us at support@intelehealth.io and we will respond in a reasonable time. Please note that Intelehealth is a Personal Information Processor, and we will have to coordinate access or deletion requests with the Personal Information Controller, Department of Health, Government of Philippines.
+To make any requests for access, modification, or deletion of your personal information with us, please contact Soumyadipta Charya, at support@intelehealth.io
```

Comment:
"To make any requests for access, modification, or deletion of your personal information with us, please contact the Data Privacy Officer, at support@intelehealth.io"

Reply:
Also this should be an app string so that it can be translated into multiple languages

Reply:
> 
> 
> "To make any requests for access, modification, or deletion of your personal information with us, please contact the Data Privacy Officer, at [support@intelehealth.io](mailto:support@intelehealth.io)"

Done! made the changes.